data/cc-collect-data.sh.in

```diff
@@ -164,6 +164,7 @@ show_log_details()
 
 	show_runtime_log_details
 	show_proxy_log_details
+	show_shim_log_details
```

Comment:
A query - did you manage to get shim debug/details into the logs? I recently saw we have a debug ability in the shim:
https://github.com/clearcontainers/shim/blob/5da156955ebb4c001b5f89f1de11db3ca916228e/src/shim.c#L992
but could find no way for us to invoke that through the runtime/proxy code. If there is a way, we should add it to the debugging docs that we are building.
/cc @amshinde

Reply:
Ugh - I've just realised the code here is wrong for the shim. I'll raise a follow-up to resolve that.

Regarding enabling debug for `cc-shim`, I've just raised https://github.com/containers/virtcontainers/pull/402. Once that lands we can re-vendor the runtime and add a new option in the config file for shim debug.

Reply:
Fixed on #682.